### PR TITLE
EPMRPP-37787 || Attachments removing impl api

### PR DESCRIPTION
**Pull request by Yumfriez**

### CODE REVIEW

File: src/main/java/com/epam/ta/reportportal/core/log/impl/CreateLogHandler.java [hunk 1]
```diff
@@ -75,6 +78,8 @@ public void setDataStoreService(DataStoreService dataStoreService) {
 
 	@Override
 	@Nonnull
+
+	//TODO check saving an attachment of the item of the project A in the project's B directory
```
Comment (pbortnik) [inline src/main/java/com/epam/ta/reportportal/core/log/impl/CreateLogHandler.java]: ?

File: src/main/java/com/epam/ta/reportportal/ws/converter/builders/AttachmentBuilder.java [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+ */
+public class AttachmentBuilder implements Supplier<Attachment> {
+
+	private final Attachment attachment;
+
+	public AttachmentBuilder() {
+		this.attachment = new Attachment();
+	}
+
+	//TODO rename path field to FILE ID
```
Comment (pbortnik) [inline src/main/java/com/epam/ta/reportportal/ws/converter/builders/AttachmentBuilder.java]: ?

File: src/main/java/com/epam/ta/reportportal/ws/converter/builders/AttachmentBuilder.java [hunk 3]
```diff
@@ -0,0 +1,70 @@
...
+	public AttachmentBuilder withPath(String path) {
+		attachment.setPath(path);
+		return this;
+	}
+
+	//TODO rename thumbnail path field to FILE ID
```
Comment (pbortnik) [inline src/main/java/com/epam/ta/reportportal/ws/converter/builders/AttachmentBuilder.java]: ?

File: src/main/java/com/epam/ta/reportportal/ws/converter/builders/AttachmentBuilder.java (hunk 2 above)
Comment (Yumfriez) [reply src/main/java/com/epam/ta/reportportal/ws/converter/builders/AttachmentBuilder.java]: renamed

File: src/main/java/com/epam/ta/reportportal/ws/converter/builders/AttachmentBuilder.java (hunk 3 above)
Comment (Yumfriez) [reply src/main/java/com/epam/ta/reportportal/ws/converter/builders/AttachmentBuilder.java]: renamed

File: src/main/java/com/epam/ta/reportportal/core/log/impl/CreateLogHandler.java (hunk 1 above)
Comment (Yumfriez) [reply src/main/java/com/epam/ta/reportportal/core/log/impl/CreateLogHandler.java]: It's a future improvement

**Pull request merged**